<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#titleLabel {
background: white;
color: black;
font-size: 20px;
border: none;
border-bottom:  1px solid black;
padding: 5px;
}

#mainFrame {
border: none;
background: white;
}

#loginFrame {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ddf, stop: 1 #aaf);
border: 1px solid gray;
padding: 10px;
border-radius: 25px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="styleSheet">
       <string notr="true">background-color: #F2F1F0;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">background-image:url(:/images/res/images/nav-bg.jpg)</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="chatPage">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="3" rowspan="2" colspan="2">
            <widget class="QFrame" name="verticalFrame">
             <property name="styleSheet">
              <string notr="true">background: none;
background-color: #F2F1F0;
border-radius:3px;
border: 1px solid #C4C1BD;
color: #4C4C4C;
padding: 4px;
</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true">border:none;
padding: 0 0 6px 0;</string>
                </property>
                <property name="text">
                 <string>User list</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="userListWidget">
                <property name="styleSheet">
                 <string notr="true">background-color: #FFF;</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0" rowspan="2" colspan="3">
            <widget class="QFrame" name="verticalFrame">
             <property name="styleSheet">
              <string notr="true">background: none;
background-color: #F2F1F0;
border-radius:3px;
border: 1px solid #C4C1BD;
color: #4C4C4C;
padding: 4px;
</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true">border:none;
padding: 0 0 6px 0;</string>
                </property>
                <property name="text">
                 <string>Chat history</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="roomTextEdit">
                <property name="styleSheet">
                 <string notr="true">background: none;
background-color: #FFF;
border-radius:3px;
border: 1px solid #C4C1BD;
color: #4C4C4C;
padding: 4px;
</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0" colspan="5">
            <widget class="QFrame" name="horizontalFrame">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>29</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background:none</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="maximumSize">
                 <size>
                  <width>90</width>
                  <height>29</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background: none;
background-color: #F2F1F0;
border-radius:3px;
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border: 1px solid #C4C1BD;
border-right: 1px solid #AAA;
color: #4C4C4C;
padding: 4px;
</string>
                </property>
                <property name="text">
                 <string>Message</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="sayLineEdit">
                <property name="styleSheet">
                 <string notr="true">background: none;
background-color: rgb(255,255,255);
border-radius:3px;
border-top-left-radius: 0px;
border-bottom-left-radius: 0px;
border: 1px solid #C4C1BD;
border-left: none;
color: #4C4C4C;
padding: 4px;
margin-right:6px;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sayButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>29</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background: none;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(240,240,240), stop:1 rgb(255, 255, 255));
border-radius:3px;
border: 1px solid #C4C1BD;
color: #4C4C4C;
padding: 4px 12px;</string>
                </property>
                <property name="text">
                 <string>Send</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="loginPage">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="3">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>223</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>100</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>223</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>267</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="titleLabel">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background: none;
background-color: #F2F1F0;
border-radius: 3px;
color: #4C4C4C;</string>
             </property>
             <property name="text">
              <string>TALK IN CODE! </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QFrame" name="loginFrame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>320</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background: none;
background-color: #F2F1F0;
border-radius: 3px;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="horizontalSpacing">
               <number>12</number>
              </property>
              <property name="verticalSpacing">
               <number>6</number>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="styleSheet">
                 <string notr="true">color: #4C4C4C;</string>
                </property>
                <property name="text">
                 <string>Base Camp:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="serverLineEdit">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255,255,255);
border-radius:3px;
border: 1px solid #C4C1BD;
color: #4C4C4C;
padding: 4px;
margin: 0px 6px 0px 0px;
color: #4C4C4C;</string>
                </property>
                <property name="text">
                 <string>talkincode.servehttp.com</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true">color: #4C4C4C;</string>
                </property>
                <property name="text">
                 <string>Username:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="userLineEdit">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255,255,255);
border-radius:3px;
border: 1px solid #C4C1BD;
color: #4C4C4C;
padding: 4px;
margin: 0px 6px 0px 0px;
color: #4C4C4C;</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="loginButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(240,240,240), stop:1 rgb(255, 255, 255));
border-radius:3px;
border: 1px solid #C4C1BD;
color: #4C4C4C;
padding: 4px 12px;</string>
                </property>
                <property name="text">
                 <string>Login</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>serverLineEdit</tabstop>
  <tabstop>userLineEdit</tabstop>
  <tabstop>loginButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>serverLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>userLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
